### PR TITLE
Refactor/Reserved words and symbols

### PR DESCRIPTION
**Pull request by chaaals**
# Changelog
This PR contains changes on TokenType reserved words and symbols

### CODE REVIEW

File: src/lexer/token.py [hunk 1]
```diff
@@ -27,52 +27,63 @@ def __format__(self, format_spec):
...
+    EQUALITY_OPERATOR = ("==", "logical_delim")
+    INEQUALITY_OPERATOR = ("!=", "logical_delim")
+    AND_OPERATOR = ("&&", "logical_delim")
+    OR_OPERATOR = ("||", "logical_delim")
+    INCREMENT_OPERATOR = ("++", "unary")
+    DECREMENT_OPERATOR = ("--", "unary")
+    CONCATENATION_OPERATOR = ("&", "concat")  # &
 
     # GROUPING SYMBOLS
     OPEN_BRACE = ("OPEN_BRACE", "open_brace")  # {
```
Comment (HolyShaq) [inline src/lexer/token.py:89]:
```suggestion
    OPEN_BRACE = ({", "open_brace")  # {
```

**Review by HolyShaq — CHANGES_REQUESTED** [review]
Papalitan token types for these as well to match the lexeme

Tas tanggalin si negative token type

**Review by HolyShaq — COMMENTED** [review]
Change this to "-" din pala

**Comment by Gidsss** [comment]
Done bro

**Comment by am-cid** [comment]
> 
> 
> Change this to "-" din pala

should probably just remove type indicator altogether and let syntax handle it whether it's a minus sign or a type indicator

**Comment by HolyShaq** [comment]
> > 
> > Change this to "-" din pala
> 
> should probably just remove type indicator altogether and let syntax handle it whether it's a minus sign or a type indicator

Alright, that's smart. I'm honestly just reluctant to get rid of my code for that 😭😭 HAHAHAHAHAHHAA

**Pull request merged**